examples/webgl_materials_parallaxmap.html

```diff
@@ -120,7 +120,7 @@
 				const gui = new GUI();
 
 				gui.add( effectController, 'mode', Object.keys( ParallaxShader.modes ) ).onChange( guiChanged );
-				gui.add( effectController, 'scale', 0.0, 0.01, 0.001 ).onChange( guiChanged );
+				gui.add( effectController, 'scale', 0.0, 0.05, 0.001 ).onChange( guiChanged );
```

Comment:
Sorry, but I don't this new value is appropriate. We should only use values in examples such that the visual output does not look broken.

Reply:
I decreased the maximium value to `0.015`.

Reply:
I decreased the maximium value to `0.015`.

Reply:
> We should only use values in examples such that the visual output does not look broken.

Exactly.